src/Nethermind/Nethermind.State/VerkleWorldState.cs

```diff
@@ -52,6 +52,11 @@ public class VerkleWorldState : IWorldState
     // // False positives would be problematic as the code _must_ be persisted
     private readonly LruKeyCache<Hash256> _codeInsertFilter = new(2048, "Code Insert Filter");
 
+    public byte[]? GetValue(Hash256 key)
```

Comment:
here we only need this to check if the value is present or not - so we dont need to actually return the value
you can just return true and false and have a funciton like `ValuePresentInTree`
Also, you can add similar functions in the tree also, to just check presence - this will be much faster then finding and returning the value